source/speech/speech.py

```diff
@@ -1540,7 +1540,17 @@ def getTextInfoSpeech(  # noqa: C901
 				if autoLanguageSwitching and newLanguage!=lastLanguage:
 					relativeSpeechSequence.append(LangChangeCommand(newLanguage))
 					lastLanguage=newLanguage
-	if reportIndentation and speakTextInfoState and allIndentation!=speakTextInfoState.indentationCache:
+	if (
+		reportIndentation
+		and speakTextInfoState
+		and(
+			# either not ignoring blank lines
+			not formatConfig['ignoreBlankLinesForRLI']
+			# or line isn't completely blank
+			or any(isinstance(t, str) and not all(c in LINE_END_CHARS for c in t) for t in textWithFields)

```

Comment:
Isn't this a case perhaps where having this expression on one line makes things more readable? The preceding line succintly summarizes what that expression computes. I am imagining in most cases when someone is reading this part of the code they just want to understand what is happening at a high level. Spreading out this expression over multiple lines, particularly since it computes a simple and very straightforward thing means a reader has to arrow through more lines of code. When it is on one line, they can skip past it with one key stroke if they don't care about the details (probably 90% of the time) and if they do care about the details then they can move by word.

Alternatively, what if this expression was moved into a helper function where the expression was spread out more as you suggest, but that way this bit of code where it would be called would remain succinct?

Reply:
a helper function may be better here - but the current line is not parseable for me at all. I had to separate it to newlines understand the change here.